### DISCUSSION TITLE
Anaconda Installation Suggestion

### DISCUSSION THREAD

**Discussion by Zanderama** [geopandas/geopandas#2747] [Installation]
Hi, I just installed Anaconda on Windows 10 as per Geopandas recommendation.

However, trying to then install geopandas resulted in lots of _Solving environment: failed_ error messages, [as per this conda issue](https://github.com/conda/conda/issues/9367) 

It seems this can be readily fixed with the simple command below, which you could recommend as an 'optional step' in your install instructions.

`conda update --all`

Thanks

Alex

**Comment by JKornberg** [comment]
Hi @Zanderama, which version of python are you using? I'm trying to install it right now on python 3.9.4 and using conda update --all I still am unable to install with "UnsatisfiableError" "Output in format: Requested package -> Available versions"

**Comment by Zanderama** [comment]
Hi @JKornberg - I'm just using the default Python that comes with Anaconda - 3.8.8 which I guess is a little old, though perhaps newer Pythons don't play so well with conda etc?
